src/keybindings.c

```diff
@@ -560,6 +560,8 @@ static void init_default_kb(void)
 		GDK_Left, GDK_MOD1_MASK, "nav_back", _("Navigate back a location"), NULL);
 	add_kb(group, GEANY_KEYS_GOTO_FORWARD, NULL,
 		GDK_Right, GDK_MOD1_MASK, "nav_forward", _("Navigate forward a location"), NULL);
+	add_kb(group, GEANY_KEYS_GOTO_ADD_POSITION, NULL,
+		0, 0, "nav_add_position", _("Remember location in history"), NULL);
```

Comment:
Maybe this should include "current" in the sentence?  "Remember current location in history"?